BannerViewDocuments/IterableBannerViewReadme.md

```diff
@@ -0,0 +1,183 @@
+# UI Components provided by IterableSDK
+
+## IterableBannerView for iOS
```

Comment:
We will be providing the doc in the code for the first time like this. cc @bradumbaugh.
Do we want that approach?